api/src/pcapi/core/educational/repository.py

```diff
@@ -548,8 +550,10 @@ def _get_filtered_collective_bookings_query(
     event_date: date | None = None,
     venue_id: int | None = None,
     extra_joins: Iterable[sa.Column] | None = None,
+    extra_outer_joins: Iterable[sa.Column] | None = None,
 ) -> sa.orm.Query:
     extra_joins = extra_joins or tuple()
+    extra_outer_joins = extra_outer_joins or tuple()
```

Comment:
(détail / style)

```suggestion
    extra_outer_joins: Iterable[sa.Column] = (),
) -> sa.orm.Query:
    extra_joins = extra_joins or tuple()
```

... et d'ailleurs on devrait faire la même chose pour l'argument `extra_joins` qui existe déjà. Si c'est un itérable (et puisqu'on ne veut pas un comportement particulier lorsque l'on fournit la valeur `None`), autant indiquer une valeur itérable par défaut. Le tuple vide est un bon candidat.